lms/templates/basic_lti_launch/basic_lti_launch.html.jinja2

```diff
@@ -13,6 +13,6 @@
 {% block scripts %}
   {{ super() }}
   {% for url in asset_urls("frontend_apps_js") %}
-    <script async defer src="{{ url }}"></script>
+    <script defer src="{{ url }}"></script>
```

Comment:
The `async` attribute was removed here to ensure that scripts in the `frontend_apps_js` collection are executed in-order. `defer`-ed scripts are executed in order once HTML parsing completes but AFAIK `async` scripts are not.